library/std/src/sys/pal/unix/fs.rs

```diff
@@ -193,20 +193,17 @@ cfg_has_statx! {{
                 return Some(Err(err));
             }
 
-            // Availability not checked yet.
+            // `ENOSYS` might come from a faulty FUSE driver.
```

Comment:
This comment only makes sense in light of the immediately previous version. Please explain it as if the reader has encountered the first time. So: Why are you starting off by mentioning ENOSYS here? Instead say something like "Syscalls can return errors from things *other than the kernel per se*, like a filesystem driver, so we can't assume this is relevant for all files."